Formula/e/envelope.rb

```diff
@@ -0,0 +1,35 @@
+class Envelope < Formula
+  desc "Environment variables CLI tool"
+  homepage "https://github.com/mattrighetti/envelope"
+  url "https://github.com/mattrighetti/envelope/archive/refs/tags/0.3.11.tar.gz"
+  sha256 "1a378564b07e041fbf3212655e8c6442f8973080cf0698886764ce38982661cc"
+  license "Unlicense"
```

Comment:
```suggestion
  license any_of: ["MIT", "Unlicense"]
```

I also filed a PR to update the info on your side, https://github.com/mattrighetti/envelope/pull/29